src/util/encryption.ts

```diff
@@ -0,0 +1,13 @@
+import { createHmac } from "crypto";
+import { envVars } from "../config/env";
+
+const getSharedSecret = (): string => {
+	return envVars.HASH_SECRET;
+}
+
+export const createHashWithSharedSecret = (data): string => {
```

Comment:
you could make this a generic "createHash" function with a secondary parameter of `hashKey = envVars.HASH_SECRET`

Reply:
I had it as something similar to begin with, but felt since the hash is global wanted to make it explicit to avoid whoopsies

Reply:
I mean, having it in the params is making it explicit...  I'm not sure I understand the concern